README.md

```diff
@@ -1,26 +1,42 @@
 # sbr-admin-data
-A bulk loader for parsing and loading CSV files into a datastore with a supporting API for single record retrieval for a period and primary key combination.
```

Comment:
Documentation has been lost from the README on develop branch. Some of the documentation is this pull request is based on the old admin-data-api project